docs/framework/docker/console.md

```diff
@@ -0,0 +1,139 @@
+--
+title: Running Console applications in Docker
+description: Running Console applications in Docker
+keywords: .NET, container, Console, Applications, Docker
+author: spboyer
+manager: wpickett
+ms.date: 09/28/2016
+ms.topic: article
+ms.prod: .net-framework-4.6
+ms.technology: vs-ide-deployment
+ms.devlang: dotnet
+ms.assetid: 85cca1d5-c9a4-4eb2-93e6-4f878de07fd7
+---
+# Running console applications in Windows containers
+
+Console applications are used for many purposes; from simple querying of a status to long running document image processing tasks. In either case, the ability to start up and scale these applications are met with limitations of hardware acquisitions, startup times or running multiple instances.
+
+Moving your console applications to use Docker and Windows containers allows for starting these applications from clean state, perform the operation and then shutdown cleanly. This topic will show the steps needed to move a console application to a Windows based container and start it using a PowerShell script.
+
+The sample console application is a simple example which takes an argument, a question in this case, and returns a random answer. This could take a `customer_id` and process their taxes, or create a thumbnail for an `image_url` argument.
+
+In addition to the answer, the `Environment.MachineName` has been added to the end of the answer to show when running the application locally, your local machine name should be returned and when running in a Windows Container; the container session id is returned.
```

Comment:
> In addition to the answer, the Environment.MachineName has been added to the end of the answer to show when running the application locally, your local machine name should be returned and when running in a Windows Container; the container session id is returned.

It might help to split this into two sentences:

In addition to the answer, the Environment.MachineName has been added to the response to show the difference between running the application locally and in a Windows container. When running the application locally, your local machine name should be returned and when running in a Windows Container; the container session id is returned.